RockPaperScissors/RockPaperScissors.cs

```diff
@@ -6,20 +6,64 @@ class Program
     {
         public static void Main()
         {
-            Console.WriteLine("Enter hand 1:");
-            string hand1 = Console.ReadLine().ToLower();
-            Console.WriteLine("Enter hand 2:");
-            string hand2 = Console.ReadLine().ToLower();
-            Console.WriteLine(CompareHands(hand1, hand2));
+            
+            Console.WriteLine("Enter rock, paper or scissors:");
+            CompareHands();
```

Comment:
It would have been better if this you:
- had a method that fetched user input and returned it to main.
- had a separate method that generated a random hand for the computer.
- a sperate method that accepted 2 hands and returned which hand won, or indicated it was a tie.
- and then in main you put all those together to print out the results to the console.